pkg/volume/glusterfs/glusterfs_test.go

```diff
@@ -645,6 +655,42 @@ func TestParseClassParameters(t *testing.T) {
 			true, // expect error
 			nil,
 		},
+
+		{
+			"enable custom ep/svc name: customEpNamePrefix: myprefix",
+			map[string]string{
+				"resturl":            "https://localhost:8080",
+				"restauthenabled":    "false",
+				"gidMin":             "4000",
+				"gidMax":             "5000",
+				"volumetype":         "replicate:4",
+				"customEpNamePrefix": "myprefix",
+			},
+			&secret,
+			false, // expect error
+			&provisionerConfig{
+				url:                "https://localhost:8080",
+				gidMin:             4000,
+				gidMax:             5000,
+				volumeType:         gapi.VolumeDurabilityInfo{Type: "replicate", Replicate: gapi.ReplicaDurability{Replica: 4}, Disperse: gapi.DisperseDurability{Data: 0, Redundancy: 0}},
+				thinPoolSnapFactor: float32(1.0),
+				customEpNamePrefix: "myprefix",
+			},
+		},
+		{
+			"invalid customepnameprefix ( ie >26 chars) parameter",
+			map[string]string{
+				"resturl":            "https://localhost:8080",
+				"restauthenabled":    "false",
+				"gidMin":             "4000",
+				"gidMax":             "5000",
+				"volumetype":         "replicate:4",
+				"customEpNamePrefix": "myprefixhasmorethan26characters",
```

Comment:
Where is a check that the prefix is correctly cut at 26 characters?

Reply:
Added in latest patch set.